examples/cli/crud_partition.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+echo "CRUD partition script."
+HMC_HOST="Put here the HMC host"
+HMC_USER="Put here HMC user"
+CPC_NAME="Put here the name of the CPC"
+PART_NAME="TESTPARTITION"
+
+eval $(zhmc -h $HMC_HOST -u $HMC_USER session create)
+zhmc cpc list
+zhmc partition list $CPC_NAME
```

Comment:
The script depends on the CPC being in DPM mode (which is fine).
It seems to me that we should demonstrate in the script how to check for DPM mode.

For example by parsing the output of partition show in JSON (not tested):
```
out=$(zhmc -o json cpc show $CPC_NAME)
dpm=$(echo "$out" |python -c 'import json,sys; obj=json.load(sys.stdin); print obj["dpm-enabled"]')
if [[ "$dpm" != "True" ]]; then
  echo "CPC $CPC_NAME is not in DPM mode."
  exit 2
fi
```